frontend/ui/data/css_tweaks.lua

```diff
@@ -268,6 +268,16 @@ DocFragment {
                 ]],
             },
         },
+        {
+            id = "hyphenate_all_auto";
+            title = _("Allow hyphenation on all text"),
+            description = _("Allow hyphenation to happen on all text (except headings), in case the publisher has disabled it."),
```

Comment:
I'd probably say something like "occur" or "be applied" instead of "happen," but I'd be more inclined to just leave it out. (I just realized this isn't new, but either way. ;-) )
```suggestion
            description = _("Allow hyphenation on all text (except headings), in case the publisher has disabled it."),
```